src/tests/build.cmd

```diff
@@ -30,6 +30,7 @@ set "__MsbuildDebugLogsDir=%__LogsDir%\MsbuildDebugLogs"
 set __Exclude=%__RepoRootDir%\src\tests\issues.targets
 
 REM __UnprocessedBuildArgs are args that we pass to msbuild (e.g. /p:TargetArchitecture=x64)
+set __commandName=%~nx0
```

Comment:
Down in `Usage`, we need to print the command name, but we've already shifted the args by then. This will grab the file name and extension before we do any shifting.